lms/envs/common.py

```diff
@@ -340,9 +340,6 @@
     # Enable OpenBadge support. See the BADGR_* settings later in this file.
     'ENABLE_OPENBADGES': False,
 
-    # The block types to disable need to be specified in "x block disable config" in django admin.
-    'ENABLE_DISABLING_XBLOCK_TYPES': True,
```

Comment:
If you haven't already, someone from devops ought to do a quick grep in configuration_secure to remove any occurrences of this variable.